build/mac/cross-compile/run-on-macos.py

```diff
@@ -84,18 +84,25 @@ def get_remote_commands(tool, args, cwd, src_dir_on_host, keychain_pw,
     return result
 
 
-def get_commands_via_tmpfile(tool, args_on_host):
```

Comment:
@goodov these changes are new. They make it possible to cross-compile `sign_updater`. The reason they're needed is that `//chrome/updater/mac/signing/pipeline.py` invokes `pkgbuild` in a way where the destination `.pkg` file is not the last argument.

https://source.chromium.org/chromium/chromium/src/+/main:chrome/updater/mac/signing/pipeline.py;l=124?q=chrome%2Fupdater%2Fmac%2Fsigning%2Fpipeline